templates/glusterfs/0/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+glusterfs-server:
+  image: rancher/glusterfs:v0.2.1
+  volumes_from:
+    - glusterfs-data
+  labels:
+    io.rancher.container.hostname_override: container_name
+    io.rancher.sidekicks: glusterfs-daemon,glusterfs-data,glusterfs-volume-create
+  command: giddyup leader elect --proxy-tcp-port=2160 /opt/rancher/peerprobe.sh ${network_mode}
+glusterfs-daemon:
+  image: rancher/glusterfs:v0.2.1
+  net: ${network_mode}
+  cap_add:
+    - SYS_ADMIN
+  volumes_from:
+    - glusterfs-data
+  labels:
+    io.rancher.container.dns: 'true'
+    io.rancher.container.network: 'true'
+    io.rancher.container.hostname_override: container_name
+    io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/glusterfs-server
+  command: glusterd -p /var/run/gluster.pid --no-daemon --log-file=-
```

Comment:
Sorry, I though the `-` for a log file was an error. Good on you for figuring out thats how to get logs to stdout!